test/autoprefixer.test.js

```diff
@@ -574,16 +574,16 @@ describe('hacks', () => {
       .map(i => i.toString())
       .filter(str => str.includes('grid-template')))
       .toEqual([
-        'autoprefixer: <css input>:135:3: Autoprefixer cannot find ' +
+        'autoprefixer: <css input>:139:3: Autoprefixer cannot find ' +
         'a grid-template containing grid-area ' +
```

Comment:
Need to add "the duplicate" here.

> ... a grid-template containing **the duplicate** grid-area...

It hints at the fact that the warning is being triggered due to a duplicate area name.